wsl-builder/release-info/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"errors"
+	"os"
+
+	"github.com/0xrawsec/golang-utils/log"
```

Comment:
This import seems suspicious. Are you sure it is what you intended it to be? I don't see a lot of activity in that repository.

Reply:
Fair enough, I replaced it with `Fprintf` since we don't really need a log for such small executable.

Reply:
Yeah, I think we don’t really need log here. Once this proposal is settled (https://go.googlesource.com/proposal/+/master/design/56345-structured-logging.md), we will be able to standardize around it for all our packages and binaries.